### PR TITLE
Adjust some more ux styles

### PR DESCRIPTION
**Pull request by pablovelezvidal**
#### Summary
More UX enhancements after zoom call with @michaelgamble.
Summary of changes:
- Open accordion when clicking on chips
- Adjust spacing in accordion cards content to be 24px from left, and from top-botton should be 32px.
- Remove top margin from card content h5.
- Chevron icon to be vertically aligned centered and has a size of 24px.
- Change variable name
- Accordion card item icon when is a warning should use the sys-away-indicator color.

Also as part of this PR, I added the logic to show the success texts description to the accordion card header.
Captura de pantalla 2022-02-11 a las 1 11 40

#### Ticket Link
n/a

#### Related Pull Requests
n/a

#### Release Note
```release-note
NONE
```

### CODE REVIEW

**Review by mkraft — APPROVED** [review]

**Comment by mkraft** [comment]
@cwarnermm There are some new translations going in with this PR, just pinging you in case you optionally want to review them.

**Comment by cwarnermm** [comment]
Thanks, @mkraft! I'll review these new strings.

**Review by michaelgamble — APPROVED** [review]
Looks great based on everything we reviewed yesterday! Great work!

File: components/admin_console/workspace-optimization/dashboard.data.tsx [hunk 1]
```diff
@@ -117,8 +128,18 @@ const useMetricsData = () => {
...
+            id: 'admin.reporting.workspace_optimization.configuration.title',
+            defaultMessage: 'Configuration',
+        }),
+        description: formatMessage({
+            id: 'admin.reporting.workspace_optimization.configuration.description',
+            defaultMessage: 'You have configuration problems to resolve',
+        }),
+        descriptionOk: formatMessage({
+            id: 'admin.reporting.workspace_optimization.configuration.descriptionOk',
+            defaultMessage: 'You seem to have good configuration for SSL and Session Lengths!',
```
Comment (cwarnermm) [inline components/admin_console/workspace-optimization/dashboard.data.tsx:141]:
```suggestion
            defaultMessage: 'You've successfully configured SSL and Session Lengths!',
```

File: components/admin_console/workspace-optimization/dashboard.data.tsx [hunk 2]
```diff
@@ -180,6 +201,10 @@ const useMetricsData = () => {
             id: 'admin.reporting.workspace_optimization.access.description',
             defaultMessage: 'Web server settings could be affecting access.',
         }),
+        descriptionOk: formatMessage({
+            id: 'admin.reporting.workspace_optimization.access.descriptionOk',
+            defaultMessage: 'Access to your workspace seems available!',
```
Comment (cwarnermm) [inline components/admin_console/workspace-optimization/dashboard.data.tsx:206]:
```suggestion
            defaultMessage: 'Your web server configuration is passing a live URL test!',
```

File: components/admin_console/workspace-optimization/dashboard.data.tsx [hunk 3]
```diff
@@ -270,6 +299,10 @@ const useMetricsData = () => {
             id: 'admin.reporting.workspace_optimization.data_privacy.description',
             defaultMessage: 'Get better insight and control over your data.',
         }),
+        descriptionOk: formatMessage({
+            id: 'admin.reporting.workspace_optimization.data_privacy.descriptionOk',
+            defaultMessage: 'You seem to be using data retention and compliance features!',
```
Comment (cwarnermm) [inline components/admin_console/workspace-optimization/dashboard.data.tsx:304]:
```suggestion
            defaultMessage: 'You've enabled data retention and compliance features!',
```

File: components/admin_console/workspace-optimization/dashboard.data.tsx [hunk 4]
```diff
@@ -319,6 +352,10 @@ const useMetricsData = () => {
             id: 'admin.reporting.workspace_optimization.ease_of_management.description',
             defaultMessage: 'We have suggestions that could make your managemenet easier.',
         }),
+        descriptionOk: formatMessage({
+            id: 'admin.reporting.workspace_optimization.ease_of_management.descriptionOk',
+            defaultMessage: 'For your current usage scale, manually managing users and guests seems apropriate!',
```
Comment (cwarnermm) [inline components/admin_console/workspace-optimization/dashboard.data.tsx:357]:
```suggestion
            defaultMessage: 'Based on current usage, manually managing users and guests manually is a good choice!',
```

File: components/admin_console/workspace-optimization/dashboard.data.tsx [hunk 5]
```diff
@@ -70,8 +71,18 @@ const useMetricsData = () => {
...
+            id: 'admin.reporting.workspace_optimization.updates.title',
+            defaultMessage: 'Server Updates',
+        }),
+        description: formatMessage({
+            id: 'admin.reporting.workspace_optimization.updates.description',
+            defaultMessage: 'You have an update to consider',
+        }),
+        descriptionOk: formatMessage({
+            id: 'admin.reporting.workspace_optimization.updates.descriptionOk',
+            defaultMessage: 'Your workspace is completely up to date!',
```
Comment (cwarnermm) [inline components/admin_console/workspace-optimization/dashboard.data.tsx:84]:
```suggestion
            defaultMessage: 'Your workspace is using the latest version of Mattermost!',
```

File: components/admin_console/workspace-optimization/dashboard.data.tsx [hunk 6]
```diff
@@ -70,8 +71,18 @@ const useMetricsData = () => {
     };
 
     const getUpdatesData = (data: UpdatesParam) => ({
-        title: formatMessage({id: 'admin.reporting.workspace_optimization.updates.title', defaultMessage: 'Updates and Errors'}),
-        description: formatMessage({id: 'admin.reporting.workspace_optimization.updates.description', defaultMessage: 'You have an update to consider'}),
+        title: formatMessage({
+            id: 'admin.reporting.workspace_optimization.updates.title',
+            defaultMessage: 'Server Updates',
```
Comment (cwarnermm) [inline components/admin_console/workspace-optimization/dashboard.data.tsx:76]: Let's change "Updates" to "updates" if all other sections of this page are also sentence case.

**Review by cwarnermm — COMMENTED** [review]
I've provided text feedback for the latest strings; however, I didn't flag every instance of a given string proposal. It will be necessary to ensure that a given string is updated throughout the product code where applicable. Thanks!

File: components/admin_console/workspace-optimization/dashboard.data.tsx (hunk 6 above)
Comment (mkraft) [inline components/admin_console/workspace-optimization/dashboard.data.tsx:76]:
```suggestion
            defaultMessage: 'Server updates',
```

Comment (mkraft) [inline components/admin_console/workspace-optimization/dashboard.data.tsx]:
```suggestion
            defaultMessage: 'Server updates',
```

**Comment by mkraft** [comment]
Thanks @cwarnermm that's really great.

File: i18n/en.json [hunk 7]
```diff
@@ -1835,10 +1837,12 @@
   "admin.reporting.workspace_optimization.cta.startTrial": "Start Trial",
   "admin.reporting.workspace_optimization.cta.upgradeLicense": "Upgrade to Enterprise",
   "admin.reporting.workspace_optimization.data_privacy.description": "Get better insight and control over your data.",
+  "admin.reporting.workspace_optimization.data_privacy.descriptionOk": "You seem to be using data retention and compliance features!",
   "admin.reporting.workspace_optimization.data_privacy.retention.description": "Organizations in highly regulated industries require more control and insight with their data. We recommend starting an Enterprise trial to take advantage of Mattermost Data Retention and Compliance features.",
   "admin.reporting.workspace_optimization.data_privacy.retention.title": "Become more data aware",
   "admin.reporting.workspace_optimization.data_privacy.title": "Data Privacy",
   "admin.reporting.workspace_optimization.ease_of_management.description": "Make it easier to manage your Mattermost workspace.",
+  "admin.reporting.workspace_optimization.ease_of_management.descriptionOk": "For your current usage scale, manually managing users and guests seems apropriate!",
```
Comment (michelengelen) [inline i18n/en.json:1845]: this differs from the defaultMessage in code. is that intentional?

```suggestion
  "admin.reporting.workspace_optimization.ease_of_management.descriptionOk": "For your current usage scale, manually managing users and guests seems appropriate!",
```

**Review by michelengelen — CHANGES_REQUESTED** [review]
it seems as if the translations in `en.json` differ from the defaultMessage properties in code. yould you re-check those (although I suppose that @cwarnermm might have things to contribute here as well).

**Comment by pablovelezvidal** [comment]
@michelengelen as discussed during today's meeting, we need these changes in the main branch so QA and UX can test everything in a single branch. If you have anything else, I can address it in a different/new PR.
@mkraft @michaelgamble @furqanmlk ^^

**Pull request merged**